.github/workflows/pr-stats.yml

```
@@ -83,8 +83,8 @@ jobs:
         uses: peter-evans/create-pull-request@v6
         with:
           token: ${{ secrets.PAT_TOKEN }}
-          commit-message: "Update PR data $(date +%Y-%m-%d)"
-          title: "Update PR data $(date +%Y-%m-%d)"
+          commit-message: "Update PR data ${{ github.event.schedule == '0 0 2 * *' && 'Monthly' || 'Daily' }} collection ${{ format('{0:yyyy-MM-dd}', github.event.schedule) }}"
```

actionlint (1.7.4)

format string "{0:yyyy-MM-dd}" does not contain placeholder {0}. remove argument which is unused in the format string

(expression)